programs/sbf/Cargo.toml

```diff
@@ -157,7 +157,7 @@ solana-sbf-rust-realloc-invoke-dep = { path = "rust/realloc_invoke_dep", version
 solana-sbpf = "=0.13.0"
 solana-sdk-ids = "=3.0.0"
 solana-secp256k1-recover = "=3.0.0"
-solana-sha256-hasher = { version = "=3.0.0", features = ["sha2"] }
+solana-sha256-hasher = { version = "=3.0.1", features = ["sha2"] }
```

Comment:
In regards to the bump here, the code that was changed is inside this guard so no impact to on chain programs:
https://github.com/anza-xyz/solana-sdk/blob/25b15adcde64c354310152abb25be60ca976ea76/sha256-hasher/src/lib.rs#L21